### PR TITLE
Initlal monorepo setup

### PR DESCRIPTION
**Pull request by justinfagnani**
This sets up the monorepo with Wireit, prettier, eslint, and two package shells (tools and validator).

PTAL @43081j @rictic < until you accept invites to the repo

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
```
Comment (43081j) [inline .eslintrc.json]: AFAIK the latest `recommended` now includes `eslint-recommended` and will also set the parser, sourceType and plugin

Comment (justinfagnani) [reply .eslintrc.json]: Do you mean the latest `plugin:@typescript-eslint/recommended`?

Comment (43081j) [reply .eslintrc.json]: Yes sorry, just realised how unclear that was! 😂 They also publish a strict config now too but it needs type information

Comment (justinfagnani) [reply .eslintrc.json]: Fixed!

**Review by 43081j — APPROVED** [review]

**Pull request merged**